tests/unit/useSubStepTest.tsx

```diff
@@ -93,3 +100,191 @@ describe('useSubStep hook', () => {
         expect(newComponentToRender).toBe(MockSubStepComponent2);
     });
 });
+
+describe('useSubStep hook with skipSteps', () => {
```

Comment:
Thanks for adding tests!
Can we please document these tests with Given... When... Then... format https://github.com/Expensify/App/tree/main/tests#documenting-tests